src/benchq/resource_estimation/default_pipelines.py

```diff
@@ -319,5 +312,6 @@ def estimate_full_graph_size(
         )
     else:
         full_graph_size += algorithm_implementation.program.n_rotation_gates
+        full_graph_size += algorithm_implementation.program.c_gates
```

Comment:
question: doesn't this mean than the number of c_gates is only double counted when we do the delayed gate synthesis? Sorry I must have misspoke. It would be better to just multiply line 301 by 2 instead.

Reply:
Actually, we should have a quick chat about this when you get the chance. So just hmu when you're ready.

Reply:
Also, we should rename `full_graph_size` to `graph_complexity`.

Reply:
Add a quick comment saying that `graph_complexity` is roughly the number of `remove_sqs` operations one needs to do. Point them to ruby slippers for more details.